tests/test_pr_comments.py

```diff
@@ -123,6 +130,65 @@ def test_community_pr_comment_no_author(app):
     assert "can't start reviewing your pull request" in comment
     assert not comment.startswith((" ", "\n", "\t"))
 
+def test_community_pr_comment_no_author_cla_has_committers_cla(app):
+    pr = make_pull_request(user="FakeUser")
+    jira = make_jira_issue(key="FOO-1")
+    people = get_people_file()
+    committers = make_committers(["mduboseedx", "nedbat"])
+    with app.test_request_context('/'):
+        comment = github_community_pr_comment(pr, jira, people, committers)
+    assert "[FOO-1](https://openedx.atlassian.net/browse/FOO-1)" in comment
+    assert "can't start reviewing your pull request" in comment
+    assert "commit authors will also need to submit a signed contributor" not in comment
```

Comment:
This is where the sentence seems wrong: all these commit authors have a CLA, but it says "commit authors will also need to submit".